src/test/resources/application-unittest.properties

```diff
@@ -1,6 +1 @@
 datarepo.testWithEmbeddedDatabase=true
-
-usermetrics.appId=testapp
-usermetrics.bardBasePath=https://bard.com
-usermetrics.metricsReportingPoolSize=10
-usermetrics.syncRefreshIntervalSeconds=3600
```

Comment:
Initially when I tried to remove these parameters, several unrelated tests failed, which is what clued me into the fact that some of our controller tests were unwittingly talking to Bard.